modules/optimization/src/functions/ParsedOptimizationFunction.C

```diff
@@ -22,42 +22,42 @@ ParsedOptimizationFunction::validParams()
       "Function used for optimization that uses a parsed expression with parameter dependence.");
 
   params.addRequiredCustomTypeParam<std::string>(
-      "value", "FunctionExpression", "The user defined function.");
+      "expression", "FunctionExpression", "The user defined function.");
   params.addRequiredParam<std::vector<std::string>>(
-      "param_vars", "Names of parameters in 'value' being optimized.");
+      "param_names", "Names of parameters in 'expression' being optimized.");
```

Comment:
Do you think that this would be confusing with the `param_vector_name`? Maybe `param_symbol_names`?

Reply:
still better than vars
do you prefer that?

Reply:
Yes. It will help me to remember what it is.